src/components/Recommend/RecommendModal.style.ts

```diff
@@ -94,6 +104,30 @@ export const Tabs = styled(MuiTabs)`
 
 export const LevelDescription = styled(MarkdownBody)`
   padding: ${theme.spacing(3)}px;
-  height: 450px;
+  height: 260px;
   overflow-y: auto;
 `;
+
+export const ContentLink = styled.a`
+  font-family: Roboto;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0em;
+  text-align: left;
+  color: #00bfea;
```

Comment:
nit: I think this is meant to be our blue (`COLOR_MAP.BLUE`). for some reason its sometimes #00bfea in the mocks, but that blue is slightly different